staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -128,6 +128,20 @@ type ClientConfigLoadingRules struct {
 	// WarnIfAllMissing indicates whether the configuration files pointed by KUBECONFIG environment variable are present or not.
 	// In case of missing files, it warns the user about the missing files.
 	WarnIfAllMissing bool
+
+	// Warner is the warning log callback to use in case of missing files.
+	Warner WarningHandler
+}
+
+// WarningHandler allows to set the logging function to use
+type WarningHandler func(...interface{})
+
+func (handler WarningHandler) Warn(message string) {
+	if handler == nil {
+		klog.V(1).Info(message)
```

Comment:
what is the difference in output between klog.Warningf and klog.V(1).Info?

Reply:
`klog.Warning` prints;
```
W0524 09:11:16.915498   42167 loader.go:142] Config not found:
````
`klog.V(1).Info` prints(only after --verbosity >= 1);
```
I0524 08:48:15.896816   40664 loader.go:142] Config not found:
```